tools/test/audio/std_utils/thdnf_test_measure.m

```diff
@@ -7,18 +7,14 @@
 test.ph = [];
 test.fh = [];
 
-if isempty(test.thdnf_mask_f)
-	if ~isempty(test.thdnf_max)
-		test.thdnf_mask_f = [1 test.fs/2]; % Start from 1 due to log()
-		test.thdnf_mask_hi = test.thdnf_max * [1 1];
-	end
-else
-	if ~isempty(test.thdnf_max)
-		error('Set either thdnf_max or thdnf_mask_f & thdnf_mask_hi but not both');
-	end
-	if isempty(test.thdnf_mask_hi)
-		error('thdnf_mask_hi must be set when thdnf_mask_f is defined');
-	end
+if ~isempty(test.thdnf_max)
+    test.thdnf_mask_f = [1 test.fs/2]; % Start from 1 due to log()
+    test.thdnf_mask_hi = test.thdnf_max * [1 1];
+end
+if isempty(test.thdnf_max)
+    error('Set either thdnf_max or thdnf_mask_f & thdnf_mask_hi but not both');
+elseif isempty(test.thdnf_mask_hi)
+    error('thdnf_mask_hi must be set when thdnf_mask_f is defined');
```

Comment:
There's many different changes in this commit "Audio: BugFix : Enhancing audio quality tests for device testing". I'd split this further to topics (process_test, perf_test, thd+n, ...) and explain in commit message why it's done.

This change needs an own commit and explanation. To me it looks like 2nd if is wrong. What is the problem that is being fixed here?

Reply:
Ahaah, Good catch . I will make the correction, This should be `if` instead of `elseif`